MAINTAINERS.md

```diff
@@ -0,0 +1,74 @@
+## Maintainers Guide
+
+This guide is intended for maintainers — anybody with commit access to one or
+more Developer Journey repositories.
+
+## Methodology:
+
+A master branch. This branch MUST be releasable at all times. Commits and
```

Comment:
This is different from how we have maintained this repository up until now, though I'm fine changing how we do this.

Reply:
We don't follow that either. I deleted the section and discussing with team. Normal open source github practices should be followed.